rpcserverhandlers_test.go

```diff
@@ -1919,6 +1924,223 @@ func TestHandleGetBestBlockHash(t *testing.T) {
 	}})
 }
 
+func TestHandleGetBlockchainInfo(t *testing.T) {
+	hash, _ := chainhash.NewHashFromStr("00000000000000001e6ec1501c858506de1de47" +
```

Comment:
I'd suggest using `mustParseHash` for these.  It probably isn't defined in the tests for this module yet, but you can find it in at least `blockchain` and `txscript`.  It helps detect bad hashes in the source code and such.

Reply:
👍, updated.